optuna/study.py

```diff
@@ -733,8 +733,9 @@ def objective(trial):
 
         sampler:
             A sampler object that implements background algorithm for value suggestion.
-            If :obj:`None` is specified, :class:`~optuna.samplers.TPESampler` is used
-            as the default. See also :class:`~optuna.samplers`.
+            If :obj:`None` is specified, :class:`~optuna.samplers.TPESampler` is used for a
+            single-objective study and :class:`~optuna.samplers.NSGAIISampler` for a
+            multi-objective study. See also :class:`~optuna.samplers`.
```

Comment:
To align other code bases, how about the following?
```suggestion
            If :obj:`None` is specified, :class:`~optuna.samplers.TPESampler` is used during
            single-objective optimization and :class:`~optuna.samplers.NSGAIISampler` during
            multi-objective optimization. See also :class:`~optuna.samplers`.
```